src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -109,6 +109,10 @@ internal class RpcFunctionInvocationDispatcher : IFunctionInvocationDispatcher
 
         private async Task<int> GetMaxProcessCount()
         {
+            if (_environment.IsMultiLanguageRuntimeEnvironment())
+            {
+                return 1;
```

Comment:
do we need check here if Lang metadata is used before making the max process count for Lang worker to 1. Just a thought?

Reply:
This is like a configuration value which is set at constructor level, Thus, it's not really possible to make a lot of decisions.